packages/core/src/base-task.ts

```diff
@@ -34,7 +34,11 @@ export default abstract class BaseTask {
   get enabled() {
     this._parseConfig();
 
-    return this._enabled;
+    let enabledViaCli = this.context.cliFlags.task?.includes(this.fullyQualifiedTaskName) || false;
```

Comment:
I believe this can be enabled via the node API too. Can we call this something else to better capture that?

```suggestion
    let enabledViaFlag = this.context.cliFlags.task?.includes(this.fullyQualifiedTaskName) || false;
```